pkg/networkservice/common/connect/DESIGN.md

```diff
@@ -27,7 +27,7 @@ calls a 'cancel' function provided to it at its construction.
 
 ## connectServer
 
-connectServer keeps clientsByID [clientmap.Map](https://github.com/networkservicemesh/sdk/blob/master/pkg/tools/clientmap/gen.go#L27) mapping incoming server Connection.ID
```

Comment:
Do we need this change in `DESIGN.md`?

Reply:
This PR is not changing connect server or any logic. So why?

Reply:
Fixed.